lib/line/bot/request.rb

```diff
@@ -42,8 +45,15 @@ def payload
 
       # @return [Hash]
       def header
+        content_type =
+          if file.is_a? File
+            MIME::Types.type_for(file.path).first.to_s
```

Comment:
It would be better `'application/json;` instead because reducing the effect of other modules.

Reply:
Thank you for your advice.
Remove 'mime-types' https://github.com/line/line-bot-sdk-ruby/pull/65/commits/d4a3bcb9c9c8f2f8f2162dab37f47d869047c289